docs/Server/API/api.md

```diff
@@ -5,16 +5,21 @@ dashboard as well as any other web clients.
 
 The Pbench Server provides a set of HTTP endpoints to manage user
 authentication and curated performance information, called "dataset resources"
-or just "datasets".
+or just "datasets". A dataset represents the aggregation of server artifacts
```

Comment:
This sentence has an antecedent problem.  I suggest splitting it, e.g.:
> The Pbench Server provides a set of HTTP endpoints to manage user authentication and curated performance information.  Performance information is stored as a "dataset resource",
or just a "dataset", consisting of the aggregation of server artifacts associated with the results of a single performance experiment, for example the outputs of a benchmark workload, such as `pbench-fio` or `pbench-user-benchmark`, together with the tool data collected during the run.

Also, consider reworking the breaks between this paragraph and the previous and subsequent ones.